modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -0,0 +1,29 @@
+.download.raw.met.module <- function(dir, met, register, machine, start_date, end_date, con, 
+                                     input_met, new.site=NULL, host, overwrite=FALSE) {
+  outfolder  <- file.path(dir, met)
+  pkg        <- "PEcAn.data.atmosphere"
+  fcn        <- paste0("download.",met)
+  
+  if(register$scale=="regional") { 
+    # Right now this only means NARR but will need to be generalized once we have more 
+    # regional met products
+     raw.data.site.id <- ifelse(met %in% c("CRUNCEP", "GFDL"), new.site$id, register$siteid)
```

Comment:
OK, moved all hacks up to the top-level met.process function as suggested. I think you're right that they need to be reworked to be less hacky (read from the register, and/or query the db). But I'm really not the right person to tackle that, at least with the very limited hrs I've got left to devote on the current project. Let's figure out some funds to hire me for more PEcAn refactoring!